src/Descriptors/Schema/Schema.php

```diff
@@ -524,7 +539,9 @@ public function relationshipLinks(
         string $type,
     ): OASchema {
         $name = Str::dasherize(
-            Str::plural($relation->relationName())
+            Str::plural(method_exists($relation, 'relationName')
```

Comment:
There's no interface for this method?

Reply:
I looked into it and it's not in the Interface. This code is a remnant of the previous author so I don't know why it was done like this.

Reply:
I noticed that there would never be a relationName method on the class. The base class nor the interface have it included. I have thus removed is